color-picker-project/src/components/colorpicker/ColorPicker.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState, Fragment } from 'react';
+import { hsvaToHex } from '@uiw/color-convert';
+import { hsvaToRgbaString } from '@uiw/color-convert';
+import Wheel from '@uiw/react-color-wheel';
+import ShadeSlider from '@uiw/react-color-shade-slider';
+import Alpha from '@uiw/react-color-alpha';
+import Material from '@uiw/react-color-material';
+import EditableInputHSLA from '@uiw/react-color-editable-input-hsla';
+import './ColorPicker.css'
+
+function ColorPicker() {
+
+  const [hsva, setHsva] = useState({ h: 214, s: 43, v: 90, a: 1 });
+
+  return (
+    <div className="color-picker-div">
+      <Fragment>
+        <div className="color-picker-container">
+          <h1 style= {{ color: hsvaToHex(hsva) }} className="color-picker-title">Choose a color...</h1>
```

Comment:
The h1 should be the header for the entire app, so this should be an h2 at most (though if we use the "Main Section" from the wireframe as an h2 then this would be an h3).

We need to re-think changing the title to the selected color because some colors will have very low contrast with the background; see screenshot below.  I do wonder if there's a way to dynamically set the background color to something dark when the selected color goes past a certain threshold.

<img width="733" alt="image" src="https://github.com/chingu-voyages/v49-tier2-team-18/assets/106836794/75376f5f-a5ac-4c06-936b-e3dffa567a17">

Reply:
Ok, I can change that to h2 and make it black.